endpoint-service.tf

```diff
@@ -0,0 +1,12 @@
+resource "aws_vpc_endpoint_service" "tarball_ingester" {
+  acceptance_required        = false
```

Comment:
Given what this thing is accepting, I think this needs to be `true`.

We can probably leave it as `false` for now pending confirmation from Jim B, but will want an answer before UC connect to us.